Cities/.gitignore

```diff
@@ -0,0 +1,2 @@
+MyEnv
+
```

Comment:
Did the environment from VScode get stored in this note? We should check because it appears as though it is saved from my perspective but the note has not been pulled in. Also, I am checking to see if this review adds a review on my profile given collaborator status. No need to approve/disapprove.